pom.xml

```diff
@@ -22,6 +22,8 @@
 		<lombok-mapstruct-binding.version>0.2.0</lombok-mapstruct-binding.version>
 		<io.jsonwebtoken.jjwt.version>0.11.5</io.jsonwebtoken.jjwt.version>
 		<org.redisson.redisson-spring-boot-starter>3.23.4</org.redisson.redisson-spring-boot-starter>
+		<junit.version>4.13.1</junit.version>
+		<mockito-core.version>3.11.2</mockito-core.version>
```

Comment:
if not mistaken, we don't need these dependencies as well. because they have to be come with starter dependency of test